python/paddle/distributed/auto_parallel/constants.py

```diff
@@ -132,3 +132,9 @@ def set_field_default_config(category, field, default_value):
 DATASET = "dataset"
 set_field_default_config(DATASET, "enable", False)
 set_field_default_config(DATASET, "num_shards", 1)
+
+#########################################
+# passes configuration
+#########################################
+PASSES = "passes"
```

Comment:
Please renname `PASSES` to `OP_FUSION` since other configurations such as `AMP` and `QAT` are also passes. Maybe it is better to add a field for each type of fusions, for example `set_field_default_config(OP_FUSION, "fused_linear", False)`.